### ISSUE TITLE
I can't install on Siemens IOT2040 (Node-red@1.0.4)

### ISSUE THREAD

**Issue by elyancorrea** [netsmarttech/node-red-contrib-opc-da#11]
When I tried to install via Pallete Manager got this error:

-----------------------------------------------------------
2020-04-07T14:22:37.004Z Install : node-red-contrib-opc-da 1.0.3

2020-04-07T14:22:38.103Z npm install --no-audit --no-update-notifier --save --save-prefix="~" --production node-red-contrib-opc-da@1.0.3
2020-04-07T14:26:32.116Z [err] npm
2020-04-07T14:26:32.134Z [err]  ERR! code ETARGET
2020-04-07T14:26:32.204Z [err] npm
2020-04-07T14:26:32.209Z [err]  
2020-04-07T14:26:32.219Z [err] ERR! notarget No matching version found for node-red-node-intel-gpio@1.0.0-iot2000
2020-04-07T14:26:32.219Z [err] npm ERR!
2020-04-07T14:26:32.223Z [err]  
2020-04-07T14:26:32.226Z [err] notarget
2020-04-07T14:26:32.239Z [err]  In most cases you or one of your dependencies are requesting
2020-04-07T14:26:32.239Z [err] npm ERR! notarget a package version that doesn't exist.
2020-04-07T14:26:32.428Z [err] 
2020-04-07T14:26:32.439Z [err] npm ERR! A complete log of this run can be found in:
2020-04-07T14:26:32.439Z [err] npm 
2020-04-07T14:26:32.442Z [err] ERR!
2020-04-07T14:26:32.445Z [err]      /home/root/.npm/_logs/2020-04-07T14_26_32_321Z-debug.log
2020-04-07T14:26:32.576Z rc=1

System:
Node-red Version: @1.0.4
Device: Simatic IOT2040
SO Image: 2.6.0

There's any way to install/run this package on my device?

**Comment by gfcittolin** [comment]
Hi, this error is not related to this node, but instead to `node-red-node-intel-gpio@1.0.0-iot2000` that is not available. Please try a clean install, and let us know if the problem persists

**Comment by elyancorrea** [comment]
Thankou @gfcittolin,

I followed this step on: https://support.industry.siemens.com/tf/ww/en/posts/problem-adding-nodes-to-node-red-with-v2-6-0-image/229076/?page=0&pageSize=10 
After this, I can install OPC-DA module normally.

However, when I try to "Test and get items" still getting a message:
"No response from Node-RED runtime" 
You know if is a problem from my install? There is a way to test opc-da module?

**Comment by steuck13** [comment]
Greetings
Could you post your Node-RED log?
Thx

**Comment by elyancorrea** [comment]
Where can i find the log you need? On Node-Red debug i got this messages on debugger:

4/7/2020, 4:17:53 PMnode: 48e9082f.0627e8
msg : string[45]
"Disconnected from the server - Unknown error!"
4/7/2020, 4:17:53 PMnode: 48e9082f.0627e8
msg : string[22]
"Trying to reconnect..."
4/7/2020, 4:18:31 PM
msg : string[45]
"Disconnected from the server - Unknown error!"

I'm using Matrikon OPC Server Simulation - on Windows 10 and a local account.

**Comment by elyancorrea** [comment]
@gfcittolin and @steuck13 
I did a fresh install of my image on device and installed node-red-contrib-opc-da, no problems until now.

When I attempt to connect do OPC server, got same error above. 

Message:
> 4/7/2020, 6:07:38 PM
> msg : string[45]
> "Disconnected from the server - Unknown error!"

**Comment by elyancorrea** [comment]
@gfcittolin @gfcittolin 
I did a fresh install of node-red + node-red-contrib-opc-da on raspberry pi running raspian and got same error. I realized that my computer runninc OPC Server is not in a domain, just in the "workgroup", does it affect behavior of node module?

**Comment by gfcittolin** [comment]
No, you should be able to connect to it even if it's not part of a domain

**Comment by elyancorrea** [comment]
@gfcittolin 
OPC Server is running on my laptop: Windows 10 Pro (no domain) Version 1909 Build 18363.752 - Static IP - MatrikonOPC Server for Simulation;

You recommend any setup to allow external access?

**Comment by steuck13** [comment]
Greetings
Currently we have a few issues on W7 and W10 environments. Fixing those is in our roadmap but for now we really don't have an alternative since we are still trying to figure out what is happening.
Thank your for your feedback.

**Comment by elyancorrea** [comment]
> Greetings
> Currently we have a few issues on W7 and W10 environments. Fixing those is in our roadmap but for now we really don't have an alternative since we are still trying to figure out what is happening.
> Thank your for your feedback.

Thank you for your reply. 
Which setup are you running your server on now? so I will set up a similar server here. After this I can help you to improve documentation.
Congratulations for your work.

**Comment by steuck13** [comment]
Sure, currently we got testing machines running NT4.0 and XP SP3. 

Your help is more than welcome
Ty

**Comment by elyancorrea** [comment]
> Sure, currently we got testing machines running NT4.0 and XP SP3.
> 
> Your help is more than welcome
> Ty

I'll test on this OS versions.
Ty

**Comment by elyancorrea** [comment]
Greetings @steuck13 and @gfcittolin 
I've done a WINXP laptop run Iconic Simulator as a  OPC Server, now I got this message when I try to connect:

![image](https://user-images.githubusercontent.com/11614990/78943843-002dbc80-7a93-11ea-85a2-adaebb79e3f7.png)

I've double checked my ip, user and password, I don't have domain, so I typed "WORKGROUP". There's any idea?

**Comment by steuck13** [comment]
Greetings
Did you configured your system's DCOM to work with authenticated users? [Here](https://www.matrikonopc.com/support/docs/MatrikonOPC-Windows-XP-DCOM-Configuration.pdf) you can see how to do a setup for 'everyone' but the procedure is the same for authenticated users.

[]'s

**Comment by elyancorrea** [comment]
Hello @steuck13 
I was unaware of this setup, I will do it later and inform the result.

Ty.
